java/daikon/suppress/NIS.java

```diff
@@ -874,7 +875,9 @@ public static void remove_suppressed_invs(PptTopLevel ppt) {
   /**
    * Returns true if the specified class is an antecedent in any NI suppression
    */
-  /*@RequiresNonNull("suppressor_map")*/
+  //Use class name in front of static field to work around Checker Framework issue 764
```

Comment:
Also, this comment makes good sense in the context of the diff but may be confusing to programmers reading the source code. I would omit it and mention the issue in the commit message rather than in a code  comment.